src/main/java/com/ProjectEureka/backend/implementaion/AnswerServiceImpl.java

```diff
@@ -0,0 +1,74 @@
+package com.ProjectEureka.backend.implementaion;
+
+import com.ProjectEureka.backend.repositories.AnswerRepository;
+import com.ProjectEureka.backend.services.AnswerService;
+import com.ProjectEureka.backend.exception.ResourceNotFoundException;
+import com.ProjectEureka.backend.models.Answer;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@Transactional
+public class AnswerServiceImpl implements AnswerService {
+    @Autowired
+    private AnswerRepository answerRepository;
+
+    @Override
+    public Answer createAnswer(Answer answer) {
+        return answerRepository.save(answer);
+    }
+
+    @Override
+    public Answer updateAnswer(String id, Answer answer) {
+        Optional<Answer> answerDb = this.answerRepository.findById(id);
+        if (answerDb.isPresent()) {
+            Answer answerUpdate = answerDb.get();
+
+            answerUpdate.setMedia_urls(answer.getMedia_urls());
```

Comment:
consider changing your setters and variables within the Answer model to be camel case, I believe there's annotations you can include in your model to keep these fields snake case in the db.